newdle/client/src/components/TopHeader.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {Grid, Header} from 'semantic-ui-react';
+
+import styles from '../App.module.scss';
+
```

Comment:
Shouldn't the one between the module and the components stay?

Reply:
no strong opinion... i got annoyed a few times by my editor adding new imports after the last one :D 

since we usually have only one css import maybe there's not much benefit in separating it from the rest?